content/docs/components-and-props.md

```diff
@@ -118,15 +127,16 @@ ReactDOM.render(
 );
 ```
 
-[](codepen://components-and-props/composing-components)
+[Probeer het op CodePen](codepen://components-and-props/composing-components)
 
-Typically, new React apps have a single `App` component at the very top. However, if you integrate React into an existing app, you might start bottom-up with a small component like `Button` and gradually work your way to the top of the view hierarchy.
+Normaalgesproken hebben React applicaties helemaal bovenin een enkele `App` component.
```

Comment:
```suggestion
Normaal gesproken hebben React applicaties helemaal bovenin een enkele `App` component.
```
https://www.taaltelefoon.be/normaal-gesproken-normaalgesproken

Reply:
TIL :)